core/src/components/modal/test/animations/modal.e2e.ts

```diff
@@ -0,0 +1,42 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ config, title }) => {
+  test.describe(title('card modal - animations'), () => {
```

Comment:
Can we prefix the title with `modal:` to match similar e2e tests?

I'd recommend this block to be:
```
modal: animations
```

and then then actual test case to be:
```
card modal should clean up animations on dismiss
```

Reply:
updated in 08f301167f thanks!